Say Their Names/Source/Service/DateFormatter/DateFormatterService.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  STNDateFormatter.swift
+//  Say Their Names
+//
+//  Created by Hakeem King on 6/3/20.
+//  Copyright © 2020 Franck-Stephane Ndame Mpouli. All rights reserved.
+//
+
+import Foundation
+
+protocol DateFormatterType {
+    func string(from date: Date) -> String
+}
+
+extension DateFormatter: DateFormatterType { }
+
+class DateFormatterService {
+
+    static let shared = DateFormatterService()
```

Comment:
We are going with dependency injection for the application. Can you please add this to `Service`